examples/asr/experimental/configs/config_bpe.yaml

```diff
@@ -0,0 +1,193 @@
+#model: "QuartzNet15x5"
```

Comment:
(1) This is not 15x5 model inside the config.
(2) Why it is called config_bpe when it clearly has char-based vocab?

Reply:
Fixed